packages/hyperion-react/src/IReactComponent.ts

```diff
@@ -226,30 +226,28 @@ export function init(options: InitOptions): void {
              *  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/prototype#:~:text=prototype%20property%2C%20by%20default
            */
 
-          // $FlowIgnore[prop-missing]
           const classComponentParentClass = component.prototype;
+          let classComponentParentClassParent;
 
           if (
-            classComponentParentClass instanceof ReactModule.Component ||
-            typeof classComponentParentClass?.render === 'function' || // possibly created via React.createClass
-            Object.getPrototypeOf(Object.getPrototypeOf(classComponentParentClass)) || // not a plain function, may be with some other lifecycle methods. 
-            classComponentParentClass === ReactModule.Component.prototype // in case buggy code didn't properly inherit from ReactModule.Component
+            classComponentParentClass && (
+              classComponentParentClass instanceof ReactModule.Component ||
+              typeof classComponentParentClass.render === 'function' || // possibly created via React.createClass
+              (
+                (classComponentParentClassParent = Object.getPrototypeOf(classComponentParentClass)) &&
+                Object.getPrototypeOf(classComponentParentClassParent)
```

Comment:
Was the silent error throwing when calling `Object.getPrototypeOf` on null `Object.getPrototypeOf(classComponentParentClass)`?

Reply:
Yes, exactly. There was already check for the null for other lines, except that one case.